@here/harp-mapview/lib/MapView.ts

```diff
@@ -2547,14 +2547,18 @@ export class MapView extends THREE.EventDispatcher {
     }
 
     /**
-     * Returns the screen position of the given geo coordinates.
+     * Returns the screen position of the given position.
      *
-     * @param geoPos - The geo coordinates.
+     * @param geoPos - The position as a {@link GeoCoordinate} or `Vector3` world position.
```

Comment:
declarations from other harp modules must be prefixed with the module name, e.g.
`{@link @here/harp-geoutils#GeoCoordinates}`

in case you want to reference three.js symbols you can use the following syntax:
`{@link https://threejs.org/docs/#api/en/math/Vector3 | THREE.Vector3}`